modules/ocf_kubernetes/manifests/master/loadbalancer.pp

```diff
@@ -2,6 +2,38 @@
   include ocf::firewall::allow_web
 
   $kubernetes_worker_nodes = lookup('kubernetes::worker_nodes')
+  $kubernetes_workers = $kubernetes_worker_nodes.map |$worker|
+    { [$worker, ldap_attr($worker, 'ipHostNumber')] }
+
+  $kubernetes_services =
+  [
```

Comment:
super nit: I'd just put this on the end of the previous line, the indentation makes it pretty clear what's going on anyways:

```
  $kubernetes_services = [
    'auth',
    [...]
  ]
```